include/socgraph.php

```diff
@@ -1981,24 +2006,28 @@ function get_gcontact_id($contact) {
 		return false;
 	}
 
-	if ($contact["network"] == NETWORK_STATUSNET)
+	/// @TODO backward-compatibility or old-lost code?
```

Comment:
This is intentional. We are having the connector and the native support via OStatus. for this special task the connector contacts can be handled like native contacts.